sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/EncryptionCosmosAsyncContainer.java

```diff
@@ -246,4 +400,4 @@ ItemDeserializer getItemDeserializer() {
             return input;
         }
     }
-}
+}
```

Comment:
We should have new line at the end of the file.

Reply:
thanks will address shortly.

Reply:
addressed